hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/net/NetworkTopologyImpl.java

```diff
@@ -306,6 +315,11 @@ public Node getNode(String loc) {
     }
   }
 
+  @Override
+  public InnerNode getClusterTree() {
```

Comment:
It's no need to add this function,  cluster.getNode(ROOT) will return the tree.

Reply:
Thank you @ChenSammi for the review, I have addressed the comment.